packages/design-system/src/styles/components/_UsaBanner.scss

```diff
@@ -75,30 +79,15 @@ $usa-banner-header-action-color: #005ea2;
   }
 }
 
-.ds-c-usa-banner__content {
-  font-size: 0.99816rem;
-  margin-left: auto;
-  margin-right: auto;
-  overflow: hidden;
-  padding: 0.25rem 1rem 1rem 0.5rem;
-
-  @media (min-width: $width-sm) {
-    padding-bottom: 1.5rem;
-    padding-top: 1.5rem;
-  }
-}
-
 .ds-c-usa-banner__button {
   background-color: transparent;
   border: 0;
-  border-radius: 0;
   bottom: 0;
   box-shadow: none;
   display: block;
-  font-size: 0.8rem;
-  height: auto;
+  font-size: inherit;
```

Comment:
Same with this, can we remove this style?

Reply:
Same for this.